content/docs/introducing-jsx.md

```diff
@@ -1,59 +1,59 @@
 ---
 id: introducing-jsx
-title: Introducing JSX
+title: היכרות עם JSX
 permalink: docs/introducing-jsx.html
 prev: hello-world.html
 next: rendering-elements.html
 ---
 
-Consider this variable declaration:
+הביטו על הצהרת משתנה זו:
 
 ```js
-const element = <h1>Hello, world!</h1>;
+const element = <h1>שלום, עולם!</h1>;
 ```
 
-This funny tag syntax is neither a string nor HTML.
+תחביר תג מצחיק זה הוא לא מחרוזת ולא HTML.
```

Comment:
I have a suggestion sentence, tell me what do you think about it.
תחביר התג המצחיק הזה הוא לא מחרוזת ולא HTML

Reply:
Not much of a difference, but I can see why this might sound better. I will change.